tests/workflow/test_resolve_execution_config.py

```diff
@@ -43,23 +41,19 @@ def test_metric_tensor_lightning_edge_case():
     device = qml.device("lightning.qubit", wires=2)
 
     empty_tape = qml.tape.QuantumScript([], [qml.expval(qml.Z(0))])
-    metric_tensor_tp = TransformProgram([qml.metric_tensor])
 
-    resolved_config = _resolve_execution_config(
-        execution_config, device, [empty_tape], metric_tensor_tp
-    )
+    resolved_config = _resolve_execution_config(execution_config, device, [empty_tape])
 
-    assert resolved_config.gradient_method is qml.gradients.param_shift
```

Comment:
If I'm understanding correctly, this test suite is specifically for checking the logic that we just removed? Then shall we just get rid of this `test_metric_tensor_lightning_edge_case`?

Reply:
let's instead run the tape through `qml.metric_tensor`, and then make sure the final gradient method is parameter shift.